src/api/Alert/types.ts

```diff
@@ -31,8 +32,9 @@ export enum AlertColumnType {
 
 export interface IAlertColumn {
   type: AlertColumnType;
-  columnName?: string;
   columnTitle: string;
+  columnName?: string;
+  options?: ICurrencyOptions;
```

Comment:
since the options property can be only undefined or ICurrencyOptions, lets rename this property to something more explicit like CurrencyOptions